aws/table_aws_organizations_account.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsOrganizationsAccount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_organizations_account",
+		Description: "AWS Organizations Account",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"AccountNotFoundException"}),
+			Hydrate:           getOrganizationsAccount,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listOrganizationsAccounts,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "id",
```

Comment:
Please rearrange the columns like - name, id, arn, status, email, joined_method, joined_timestamp. For top columns(first 4 to 5 columns) we provided the important/key columns and after that, we added the columns in ascending order, and the json columns will be in the last section that too should be in ascending order.

Reply:
This should probably be clarified in the standards here: https://steampipe.io/docs/develop/standards

> Key columns should appear first, then the rest added alphabetically, then "standard" columns last. Note that help (.inspect, online docs) order the columns alphabetically regardless of the order in the create table statement.

I read that as "primary keys (e.g. ID, ARN) should appear first" - it's a bit subjective as to what is most important too. Maybe a list of column names that should be prioritised here would help?

Reply:
Yes, Thanks for the suggestion, we will discuss and update this.